topics/ecology/tutorials/species-distribution-modeling/tutorial.md

```diff
@@ -0,0 +1,364 @@
+---
+layout: tutorial_hands_on
+title: "Species distribution modeling"
+questions:
+    - "How to build & evaluate a niche model, visualize and project it ?"
+objectives:
+    - "Find and download occurrences data from GBIF"
+    - "Find and download environmental data"
+    - "Process both occurrences and environmental"
+    - "Partition occurrence data"
+    - "Model a theoretical ecological niche and predict species distribution in a future climate scenario by using SDM"
+requirements:
+time_estimation: "1h"
+key_points:
+    - ""
```

Comment:
do you have any conclusions/take-home messages to add here? (and in other tutorial)